lib/tasks/backend.rake

```diff
@@ -1,4 +1,11 @@
 namespace :backend do
```

Comment:
I think `backend` in this context refers specifically to the [`Backend` model](https://github.com/alphagov/router-api/blob/master/app/models/backend.rb). I'd suggest having a `routes` namespace instead (and although probably a little much for this PR but we could also combine together [`delete_routes`](https://github.com/alphagov/router-api/blob/master/lib/tasks/delete_routes.rake) and [`dump_routes`](https://github.com/alphagov/router-api/blob/master/lib/tasks/dump_routes.rake) into it as well).

Reply:
Fair point. I've moved the task and pushed up the commit.

[delete_routes](https://github.com/alphagov/router-api/blob/master/lib/tasks/delete_routes.rake) looks _very_... hardcoded. Does anyone run these tasks anymore? It looks to me as though we can safely delete the whole file. [dump_routes](https://github.com/alphagov/router-api/blob/master/lib/tasks/dump_routes.rake) looks a bit more generic and useful and could be moved.

Reply:
Yeah, I have no idea really, I think deleting that `delete_routes` file is reasonable.

Reply:
I've found all the reassurance I need! https://github.com/alphagov/router-api/pull/95#pullrequestreview-20214326